doc/python_reference.rst

```diff
@@ -946,28 +946,6 @@ Functions that assist with decoding and model fitting:
    get_coef
 
 
-Realtime
-========
```

Comment:
We should keep this section header and add a sentence linking to `mne-tools.github.io/mne-realtime` or wherever we put the docs for this new repo.